src/pretix/api/views/item.py

```diff
@@ -77,18 +78,58 @@ class Meta:
         fields = ['subevent']
 
 
-class QuotaViewSet(viewsets.ReadOnlyModelViewSet):
+class QuotaViewSet(viewsets.ModelViewSet):
     serializer_class = QuotaSerializer
     queryset = Quota.objects.none()
     filter_backends = (DjangoFilterBackend, OrderingFilter,)
     filter_class = QuotaFilter
     ordering_fields = ('id', 'size')
     ordering = ('id',)
     permission = 'can_change_items'
+    write_permission = 'can_change_items'
 
     def get_queryset(self):
         return self.request.event.quotas.all()
 
+    def create(self, request, *args, **kwargs):
+        with request.event.lock():
+            return super().create(request, *args, **kwargs)
```

Comment:
We don't need the lock here as we do with vouchers and can therefore drop this method completely.